src/app/log-in/log-in.component.ts

```diff
@@ -0,0 +1,58 @@
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { Router } from '@angular/router';
+import { LogInModel } from '../models/log-in-model';
+import { trigger, animate, state, style, transition } from '@angular/animations';
+
+@Component({
+  selector: 'app-log-in',
+  templateUrl: './log-in.component.html',
+  styleUrls: ['./log-in.component.css'],
+})
+export class LogInComponent implements OnInit {
+  logInForm: FormGroup = new FormGroup({});
+  isError: boolean = false;
+  logInCredintials: LogInModel = {
+    username: '',
+    password: '',
+    rememberMe: false
+  };
+
+  constructor(
+    private formBuilder: FormBuilder,
+    private router: Router,
+  ) {
+  }
+
+  ngOnInit() {
+    this.logInForm = this.formBuilder.group({
+      username: ['', Validators.required],
```

Comment:
```suggestion
      username: ['', Validators.required, Validators.minLength(10),Validators.maxLength(10),]
```

national id/ رقم الهوية always consists of 10 digits

Reply:
is the array brackets unnecessary for the validators ? the code didn't work until I added them and put all validators inside of it 
like this:
`username: ['', [Validators.required, Validators.minLength(10), Validators.maxLength(10),]],`

Reply:
I accidentally removed it